### DISCUSSION TITLE
Olympic Sports channel?

### DISCUSSION THREAD

**Discussion by logicrak** [iptv-org/community#266] [Q&A] [answered]
Anyone Got Olympic sports channel?

**Comment by MapGuy11** [comment]
No we did before it was 403.

**Reply by BellezaEmporium** [reply]
I can make my way to make it work.

https://olympics.com/tokenGenerator?url=https://ott-dai-oc.akamaized.net/OC1/master.m3u8&domain=https://ott-dai-oc.akamaized.net&_ts=1627162937521

**Reply by BellezaEmporium** [reply, marked as answer]
https://iptv-all.lanesh4d0w.codes/special/olympics
